docs/variants/pc_engines/faq.md

```diff
@@ -0,0 +1,8 @@
+# FAQ
+
+## What BIOS version is compatible with a (coreboot+SeaBIOS) Entry Subscription?
+
+The next BIOS version for your APU2 motherboard under the Dasharo
+(coreboot+SeaBIOS) Entry Subscription has not yet been determined. We plan to
+conduct tests on pfSense and will likely use the latest stable version available
+at the time of testing.
```

Comment:
@Stanislaw-bnk it could be better quality. Please try the following:

**What BIOS Version Can I Install with My Dasharo (coreboot+SeaBIOS) Subscription on an apu2/3/4/6 Motherboard?**

When you subscribe to a Dasharo (coreboot+SeaBIOS) Entry Subscription for Network Appliance, the BIOS version updates available to you are determined by Dasharo's release cycle and testing scope.

1. **Upcoming BIOS Version:**
   The next BIOS version that you will be able to install will most likely be tagged as 24.02.01.01. There is a possibility that this might shift to base on coreboot 24.05, then it would be called 24.05.01, due to the recent change in the coreboot versioning scheme. For detailed insights into the versioning scheme, please refer to the coreboot release notes here: [Coreboot Release Notes](https://doc.coreboot.org/releases/coreboot-24.02-relnotes.html?highlight=releases#release-number-format-update) as well as discussion about [Dasharo Versioning](https://github.com/Dasharo/dasharo-issues/issues/762)

2. **Test Scope:**
   While the exact test scope for the upcoming BIOS version is currently being finalized, it will offer feature parity with version 4.19.0.1. The test scope for version 4.19.0.1, which might give you an idea of what to expect, can be found in this document: [Dasharo Test Scope Document](https://docs.google.com/spreadsheets/d/1_uRhVo9eYeZONnelymonYp444zYHT_Q_qmJEJ8_XqJc/edit#gid=0).

3. **Release Information:**
   Detailed information about the release, including the scope and enhancements, will be communicated through our release newsletter. Additionally, all release notes and relevant updates will be made available on the Dasharo documentation site at [Dasharo Release Documentation](https://docs.dasharo.com/variants/pc_engines/releases_seabios/).

Reply:
fixed: https://github.com/Dasharo/docs/pull/813/commits/3e4381ae44cc550aaeb4aee725b3e6ed1e807649
But there is 1 failing check due to line length of the Question.

Reply:
@Stanislaw-bnk This is not the first time when we have had to disable this kind of check: https://github.com/Dasharo/docs/blob/master/docs/osf-trivia-list/dasharo.md?plain=1#L339